### ISSUE TITLE
BUGs

### ISSUE THREAD

**Issue by uloveqian2021** [dianwen-ng/Keyword-Spotting-ConvMixer#1]
a brilliant job, But some error in code at line         

"""
with autocast():
            output = model(data)
            loss = criterion(output, target)
"""
maybe output = model(data)[0]

**Comment by dianwen-ng** [comment]
thanks for pointing out. Edited with correct unpacking.
